cypress/component/date/DateEditor.spec.tsx

```diff
@@ -112,16 +88,18 @@ describe('Date Editor', () => {
 
   describe('default configuration', () => {
     it('should read initial value', () => {
-      openPage();
-      cy.setInitialValue('2018-01-03T05:53+03:00');
+      const [fieldSdk] = createFakeFieldAPI(undefined, '2018-01-03T05:53+03:00');
+      mount(<DateEditor field={fieldSdk} isInitiallyDisabled={false} />);
 
       selectors.getDateInput().should('have.value', '03 Jan 2018');
       selectors.getTimeInput().should('have.value', '05:53');
       selectors.getTimezoneInput().should('have.value', '+03:00');
     });
 
     it('should render date, time (24 format) and timezone inputs by default', () => {
-      openPage();
+      const [fieldSdk] = createFakeFieldAPI();
```

Comment:
To avoid repetition, maybe we can extract this into a function?

```suggestion
      function setupDateEditor() {
          const [fieldSdk] = createFakeFieldAPI();
          mount(<DateEditor field={fieldSdk} isInitiallyDisabled={false} />);
      }
```

Reply:
Sure, could do that with some args.

Reply:
Although we do need to use the `fieldSdk` directly in some tests. Do you see any readability issues with returning it from the setup func?

Reply:
So it would look like this

```const setupDateEditor = (
  initialValue?: string,
  initiallyDisabled = false,
  parameters?: Parameters
) => {
  const [fieldSdk] = createFakeFieldAPI(undefined, initialValue);
  mount(
    <DateEditor field={fieldSdk} isInitiallyDisabled={initiallyDisabled} parameters={parameters} />
  );
  return fieldSdk;
};```

Reply:
I think that looks good 👍